src/OpenTelemetry.Instrumentation.Http/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased
 
+* Implementation of HTTP retries and redirects instrumentation defined in
```

Comment:
Did this spec get implemented in this PR? are we adding "http.retry_count" ? Not able to see any change related to that in this PR...

Reply:
Correct. In order to add `http.retry_count` we should propagate a context between tries (which is also required to enable retry/redirect spans correlation with links, see https://github.com/open-telemetry/opentelemetry-specification/pull/2393). 

Since span links concept and its usage is not yet clear for the community, and context propagation topic was postponed, we cannot add `http.retry_count` at this time, but it's going be added in the future.

Reply:
Thanks!
Could you update the PR description as well? Its not clear what the user can expect once they upgrade to a version with this change.